test/PandoraInterface.cxx

```diff
@@ -18,6 +18,10 @@
 #include "larpandoracontent/LArPlugins/LArPseudoLayerPlugin.h"
 #include "larpandoracontent/LArPlugins/LArRotationalTransformationPlugin.h"
 
+#ifdef LIBTORCH_DL
+#include "larpandoradlcontent/LArDLContent.h"
```

Comment:
Retrospectively adding a comment here about preprocessor formatting. Might want to indent the include.